lumicks/pylake/correlated_stack.py

```diff
@@ -299,11 +307,18 @@ def plot_correlated(
         plot_correlated(
             channel_slice,
             frame_timestamps,
-            lambda frame_idx: self._get_frame(frame_idx)._get_plot_data(channel),
+            lambda frame_idx: self._get_frame(frame_idx)._get_plot_data(
+                channel, adjustment=adjustment
+            ),
             title_factory,
             frame,
             reduce,
             figure_scale=figure_scale,
+            post_update=None
+            if adjustment is None
+            else lambda image_handle, image: adjustment._update_limits(
+                image_handle, image, channel
+            ),
```

Comment:
this is some funky formatting, I think it'd be better to just define it before the function call

- or if you go with my previous suggestion the conditional drops out anyway